src/input.c

```diff
@@ -478,7 +478,7 @@ void input_getKeybindDisplay( const char *keybind, char *buf, int len )
          if (key < 0x100 && isalpha(key))
             p += scnprintf( &buf[p], len-p, "%c", toupper(key) );
          else
-            p += scnprintf( &buf[p], len-p, "%s", _(SDL_GetKeyName(key)) );
```

Comment:
"key" is not translatable, because the POT file has no SDL key names.

Reply:
Included the key names and made them translatable in main.